x-pack/test/apm_api_integration/trial/tests/service_maps.ts

```diff
@@ -4,254 +4,284 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import querystring from 'querystring';
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../common/ftr_provider_context';
 
 export default function serviceMapsApiTests({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
   const esArchiver = getService('esArchiver');
 
-  const start = encodeURIComponent('2020-06-29T06:45:00.000Z');
-  const end = encodeURIComponent('2020-06-29T06:49:00.000Z');
+  describe('Service Maps with a trial license', () => {
+    describe('/api/apm/service-map', () => {
+      describe('when there is no data', () => {
+        it('returns empty list', async () => {
+          const response = await supertest.get(
+            '/api/apm/service-map?start=2020-06-28T10%3A24%3A46.055Z&end=2020-06-29T10%3A24%3A46.055Z'
```

Comment:
I think it's easier to read like:
```js
const response = await supertest.get(`/api/apm/service-map?start=${start}&end=${end}`);
```

Reply:
Not sure I remember why this was changed but I think the form in https://github.com/elastic/kibana/pull/69520/files/4a0a421c9873a9e90fa10c578d4f2a1a2d12dd1c#diff-4c35b4dbeb33d72f5246c5c36adc2dfdR267-R272 is better than both.